pyccel/codegen/printing/ccode.py

```diff
@@ -2835,6 +2835,20 @@ def _print_DictClear(self, expr):
         dict_var = self._print(ObjectAddress(expr.dict_obj))
         return f"{c_type}_clear({dict_var});\n"
 
+    def _print_DictGet(self, expr):
+        target = expr.dict_obj
+        class_type = target.class_type
+        c_type = self.get_c_type(class_type)
+
+        dict_var = self._print(ObjectAddress(target))
+        key = self._print(expr.key)
+
+        if expr.default_value is not None:
+            default_val = self._print(expr.default_value)
+            return f"({c_type}_contains({dict_var}, {key}) ? *{c_type}_at({dict_var}, {key}) : {default_val})"
+
+        return errors.report("Accessing dictionary using get without default is not supported yet", severity="fatal")
```

Comment:
Please provide `expr` to `errors.report` as the `symbol` argument. This ensures that the error reports line and column information:
```suggestion
        raise errors.report("Accessing dictionary using get without default is not supported yet",
                  symbol=expr, severity="fatal")
```

Reply:
Added, thanks 😄